internal/store/access_role.go

```diff
@@ -51,7 +51,7 @@ func (s *Store) CreateCloudProviderAccessRole(groupID, provider string) (*atlas.
 		result, _, err := s.client.(*atlas.Client).CloudProviderAccess.CreateRole(context.Background(), groupID, req)
 		return result, err
 	default:
-		return nil, fmt.Errorf("unsupported service: %s", s.service)
+		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
```

Comment:
we have now a sentinel error `errUnsupportedService`, all changes here is to use the new sentinel value